### ISSUE TITLE
Uncaught TypeError: crypto.randomBytes is not a function

### ISSUE THREAD

**Issue by JeongHwanMun** [aergoio/athena-ide-atom#3]
[Enter steps to reproduce:]

1. update version
2. restart atom

<img width="480" alt="스크린샷 2019-05-08 오후 5 22 46" src="https://user-images.githubusercontent.com/6779228/57360832-17df2f80-71b6-11e9-8223-d15ffdc82e96.png">

**Atom**: 1.36.1 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.4
**Thrown From**: [athena-ide-atom](https://github.com/aergoio/athena-ide-atom) package 0.3.6

### Stack Trace

Uncaught TypeError: crypto.randomBytes is not a function

```
At /Users/jeonghwan/.atom/packages/athena-ide-atom/dist/athena-ide-atom.js:14265

TypeError: crypto.randomBytes is not a function
    at nodeRNG (/packages/athena-ide-atom/dist/athena-ide-atom.js:14265:17)
    at v4 (/packages/athena-ide-atom/dist/athena-ide-atom.js:14302:52)
    at GoogleAnalytics.fs.exists.existance (/packages/athena-ide-atom/dist/athena-ide-atom.js:14402:51)
    at FSReqWrap.cb [as oncomplete] (fs.js:312:19)
```

### Commands

```

```

### Non-Core Packages

```
athena-ide-atom 0.3.6 
atom-beautify 0.33.4 
atom-format-lua 0.1.5 
busy-signal 2.0.1 
ftp-remote-edit 0.15.2 
html-template-generator 0.3.0 
intentions 1.1.5 
language-graphql 1.0.0 
language-vue 0.24.0 
linter 2.3.0 
linter-ui-default 1.7.1 
platformio-ide-terminal 2.9.1 
react-es6-snippets 0.3.0 
```

**Comment by acktsap** [comment]
Maybe fixed in 5b7c6ebbdc6aab73bb738f82e4f4742b93f605c2
But not sure, I'll close the issue after enough test

**Comment by acktsap** [comment]
I'm sure it's been fixed. It will be available soon (in v0.4.0).
